createjson.rb

```diff
@@ -0,0 +1,59 @@
+require 'json'
+
+class CreateJsonBook
+  def initialize(app)
+    @app = app
+  end
+
+  def create_file
+    my_book = {}
+    books = []
+    @app.books.each do |book|
+      single_boook = { Title: book.title, Author: book.author }
+      books.push(single_boook)
+    end
+    my_book = { Book: books }.to_json
+    open('books.json', 'a') do |file|
+      file.puts my_book
+    end
+  end
+end
+
+class CreateJsonPeople
+  def initialize(app)
+    @app = app
+  end
+
+  def create_file
+    my_person = {}
+    persons = []
+    @app.people.each do |people|
+      single_person = { Person: people, Name: people.name, ID: people.id, Age: people.age }
+      persons.push(single_person)
+    end
+    my_person = { Person: persons }.to_json
+    open('people.json', 'a') do |file|
+      file.puts my_person
+    end
+  end
+end
```

Comment:
- [ ] While the files were created, they were not read and when I ran the app again and pressed the number to read it, it maintains no data, it is because the json file was not imported in the app file, you conditionally read the @books and others but no json file imported, kindly work on reading the files.

Reply:
Can you please guide on where exactly is the problem so I can work on it?

Reply:
I am sorry for not making it clear, the problem is reading the file upon launching the app, so it is in the app file.